src/components/Import/FileImport.vue

```diff
@@ -75,8 +76,10 @@ export default Vue.extend({
 
           let key: { enc: string; hash: string } | null = null;
 
-          if (importData.key) {
-            key = importData.key;
+          if (importData.hasOwnProperty("key")) {
```

Comment:
Yes, the error with backup creates data like this:
```json
{
    "key": null
}
```
Because of that
```js
Bool(importData.key) == false
```
so the only way to check if the key exists but is not null is:
```js
importData.hasOwnProperty("key")
// or
"key" in importData
```

Reply:
I see.